vufs/src/main/java/com/stnetix/ariaddna/vufs/BusinessObjects/Block.java

```diff
@@ -0,0 +1,61 @@
+package com.stnetix.ariaddna.vufs.BusinessObjects;
+
+import de.greenrobot.common.hash.Murmur3A;
+
+/**
+ * Block it is elementary unity in ariADDna storage.
+ */
+public class Block {
+    private String mVersion;
```

Comment:
I don't understand why you use prefix "m" in the name class filelds? I recommend delete them :)

Reply:
OK